templates/hpe/index.html

```diff
@@ -105,8 +107,10 @@ <h2>Ubuntu: Tier1 operating <br /> system on HPE hardware</h2>
         }}
       </div>
     </div>
-    <div class="col-6 col-medium-3">
-      <h2>Reliable solutions <br />for projects of any size</h2>
+    <div class="col-6 col-medium-6">
+      <div class="p-block">
```

Comment:
If we need extra space after a heading that should be handled via the typography?